Userland/Libraries/LibWeb/HTML/BrowsingContext.cpp

```diff
@@ -604,8 +604,7 @@ BrowsingContext::ChosenBrowsingContext BrowsingContext::choose_a_browsing_contex
 
             // 3. If noopener is true, then set chosen to the result of creating a new top-level browsing context.
             if (no_opener == TokenizedFeature::NoOpener::Yes) {
-                auto handle = m_page->client().page_did_request_new_tab(activate_tab);
-                chosen = RemoteBrowsingContext::create_a_new_remote_browsing_context(handle);
+                chosen = nullptr;
```

Comment:
Won't we now crash below?
```c++
// 6. If name is not an ASCII case-insensitive match for "_blank", then set chosen's name to name.
if (!Infra::is_ascii_case_insensitive_match(name, "_blank"sv))
    chosen->set_name(String::from_utf8(name).release_value_but_fixme_should_propagate_errors());
```

Reply:
Yes, but this is only called from WebDriverConnection. I have a series of commits in the pipe to convert that to calling choose_a_navigable instead.

Reply:
Sure, but it'd be nice not to knowingly crash even for WebDriver. Can add a check for `if (chosen)` with a FIXME just for this PR I think